cpp/src/join/conditional_join.cu

```diff
@@ -210,29 +305,28 @@ std::size_t compute_conditional_join_output_size(table_view const& left,
   // We can immediately filter out cases where one table is empty. In
   // some cases, we return all the rows of the other table with a corresponding
   // null index for the empty table; in others, we return an empty output.
-  auto right_num_rows{right.num_rows()};
-  auto left_num_rows{left.num_rows()};
-  if (right_num_rows == 0) {
+
+  if (right.num_rows() == 0) {
```

Comment:
This is the place that @bdice was referring to https://github.com/rapidsai/cudf/pull/14646/files#r1573011742.

Depending on how you see it, we should consistently use either `auto const ...` or `.num_rows()` in both functions

Reply:
I'll change the entire file to be consistent

Reply:
Ah... Okay. It turns out i was pushing to the wrong fork. That's why there were was some inconsistencies. My bad.